showcase/app/templates/page-components/form/file-input.hbs

```diff
@@ -44,40 +44,40 @@
   <div class="{{if this.showHighlight 'shw-component-form-file-input-layout-highlight'}}">
     <Shw::Text::H3>Content</Shw::Text::H3>
 
-    <Shw::Grid @columns="3" @gap="2rem" as |SG|>
+    <Shw::Grid @columns={{3}} @gap="2rem" as |SG|>
       <SG.Item @label="Only label">
-        <Hds::Form::FileInput::Field @value="Lorem ipsum dolor" as |F|>
+        <Hds::Form::FileInput::Field value="Lorem ipsum dolor" as |F|>
```

Comment:
The `@value` probably came from a copy&paste from the `TextInput` demos. This is a File Input, the `value` attribute does nothing, we can remove it entirely.
```suggestion
        <Hds::Form::FileInput::Field as |F|>
```